Resources/Textures/_Harmony/Objects/Weapons/Guns/Ammunition/Magazines/EnergyCartridges/Carbine/jailbroken.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "license": "CC-BY-NC-SA-3.0",
+    "copyright": "Created by BoskiYourk for use in Harmony",
+    "size": {
+          "x": 32,
+          "y": 32
+    },
```

This is over-indented.
```suggestion
    "size": {
        "x": 32,
        "y": 32
    },
```